src/sagemaker/jumpstart/types.py

```diff
@@ -1170,11 +1197,14 @@ class JumpStartModelSpecs(JumpStartMetadataBaseFields):
 
     def __init__(self, spec: Dict[str, Any]):
         """Initializes a JumpStartModelSpecs object from its json representation.
+        By default only apply the inference preset config.
 
         Args:
             spec (Dict[str, Any]): Dictionary representation of spec.
         """
         self.from_json(spec)
+        if self.inference_presets and self.inference_presets.get_top_config_from_ranking():
+            super().from_json(self.inference_presets.get_top_config_from_ranking().resolved_config)
```

Comment:
im a bit confused, can you explain what's going on here?

Reply:
One `JumpStartModelSpec` has two set of `PresetConfigs` inside of it - one set for inference and one set for training. When initializing the class, we choose to apply the default inference `PresetConfig` from the overall ranking to override these fields. (Open to discussion though if we want to apply the best training config as well)

Alternatively, we can just remove this two lines of logic, so that the fields in JumpStartModelSpec stays with default base fields, until someone calls the `set` method with a config they want to apply.